psi-ms.obo

```diff
@@ -22260,6 +22260,33 @@ def: "Accession number (e.g. in PeptideAtlas) of the peptide sequence" [PSI:PI]
 is_a: MS:1003050 ! peptidoform attribute
 relationship: has_value_type xsd\:string ! The allowed value-type for this CV term
 
+[Term]
+id: MS:1003245
+name: ultraviolet photodissociation
+def: "Multiphoton ionization where the reactant ion dissociates as a result of the absorption of multiple UV photons." [PSI:MS]
+synonym: "UVPD" EXACT []
+is_a: MS:1000435 ! photodissociation
+
+[Term]
+id: MS:1003246
+name: negative electron transfer dissociation
+def: "A process to fragment ions in a mass spectrometer by inducing fragmentation of anions (e.g. peptides or proteins) by transferring electrons to a radical-cation." [DOI:10.1016/j.jasms.2005.01.015] [PSI:MS]
```

Comment:
The source of the definition should be a comma separated list enclosed in square braces:

```suggestion
def: "A process to fragment ions in a mass spectrometer by inducing fragmentation of anions (e.g. peptides or proteins) by transferring electrons to a radical-cation." [DOI:10.1016/j.jasms.2005.01.015, PSI:MS]
```